packit_service/service/api/copr_builds.py

```diff
@@ -35,6 +35,13 @@
 ns = Namespace("copr-builds", description="COPR builds")
 
 
+def if_time_exists(datetime_object):
+    if datetime_object:
```

Comment:
Although this is completely functional, just for the sake of being explicit, you could write `if datetime_object is not None`.